### DISCUSSION TITLE
How to observe extension enum

### DISCUSSION THREAD

**Discussion by estherzhangyue** [jonataslaw/getx#2929] [Q&A] [answered]
I need to observe an enum with some extension value, like below

```
enum ActivityType { init, search}

extension ActivityTypeValue on ActivityType {
  String get title {
    switch (this) {
      case ActivityType.init:
        return 'Init';
      case ActivityType.searching:
        return 'Searching...';
    }
  }
}
```
And I add obs in controller like this

`var currentStatus = ActivityType.init.obs;`

but when I wanna handle title data it throw **_The getter 'title' isn't defined for the type 'Rx<ActivityType>'_**  error, can anyone know how to add observer to enum with some extension value?

Thank you!

**Comment by jessicamrbr** [comment, marked as answer]
Did you try currentStatus.value.tittle ?

**Reply by estherzhangyue** [reply]
thx，works
